conans/client/migrations.py

```diff
@@ -75,8 +78,21 @@ def _migrate_pkg_db_lru(cache_folder, old_version):
         undo_lru = textwrap.dedent("""\
             import os
             import sqlite3
+
             def migrate(cache_folder):
-                db = os.path.join(cache_folder, 'p', 'cache.sqlite3')
+                config = os.path.join(cache_folder, "global.conf")
+                global_conf = open(config, "r").read() if os.path.isfile(config) else ""
+                storage = os.path.join(cache_folder, "p")
+                for line in global_conf.splitlines():
+                    tokens = line.split("=")
+                    if len(tokens) != 2:
+                        continue
+                    conf, value = tokens
```

Comment:
shoudln't this use ConfDefinition to parse quotes and whatnot?

Reply:
The truth is that I was trying to do a shortcut, to avoid having to check every change in the code in the previous versions, because the code has to be compatible with the old version, not that evident with all the work and changes since 2.0.0. But yeah, probably leveraging some of the previous code is more robust than this, I'll try to find some common base.

Reply:
Done